lib/rucio/db/sqla/util.py

```diff
@@ -233,11 +244,13 @@ def get_db_time():
         else:
             query = select([func.current_date()])
 
-        for now, in s.execute(query):
-            if storage_date_format:
-                return datetime.strptime(now, storage_date_format)
-            return now
-
+        # sqlAlchemy 2.0 format
+        with s() as session:
```

Comment:
again, please use the same convention everywhere. I already saw 'Session', 'session' and 's' to designate a scoped_session. I do completely agree that these things were incorrectly named at the first place, but this is a good opportunity to make things right and avoid confusion in the future.

Reply:
I will make an overall consistent check and change the code accordingly.

Reply:
Done